### PR TITLE
Volkov vadim/login

### PR DESCRIPTION
**Pull request by Trivme**
Fixed and changed negative login tests

### CODE REVIEW

File: modules/user/login/login_negativeEmptyEmail.spec.js [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-    it('should click Login button', () => {
-        LoginPage.submitBtn.click();
-        browser.pause(2000);
-    });
-
-    it('should have correct title', () => {
-        const actual = LoginPage.h1.getText();
-        const expected = 'User Login';
-        expect(actual).eq(expected);
+    it('should Login btn is disabled', () => {
```
Comment (YelenaDipon) [inline modules/user/login/login_negativeEmptyEmail.spec.js]: write button not btn

File: modules/user/login/login_negativeEmptyEmail.spec.js [hunk 2]
```diff
@@ -1,29 +1,23 @@
 import { expect } from 'chai';
 import LoginPage from '../../_PageObjects/LoginPage';
+import {student} from  './../../_data/user.data';
+import Menu from "../../_PageObjects/Menu";
 
 describe('LOGIN PAGE NEGATIVE with empty Email field and valid Password', () => {
```
Comment (YelenaDipon) [inline modules/user/login/login_negativeEmptyEmail.spec.js]: All letters must be capitalized in your describe.

Comment (YelenaDipon) [inline modules/user/login/login_negativeEmptyEmail.spec.js]: All letters must be capitalized in your describe.

File: modules/user/login/login_negativeEmptyEmailPassword.spec.js [hunk 3]
```diff
@@ -1,25 +1,25 @@
-import { expect } from 'chai';
 import LoginPage from '../../_PageObjects/LoginPage';
+import {student} from  './../../_data/user.data';
+import Menu from "../../_PageObjects/Menu";
+import Notification from "../../_PageObjects/Notification";
 
-describe('LOGIN PAGE NEGATIVE with empty Email and empty Password fields', () => {
-    it('should open login page', () => {
+describe('LOGIN PAGE NEGATIVE with not correct email entering', () => {
```
Comment (YelenaDipon) [inline modules/user/login/login_negativeEmptyEmailPassword.spec.js]: All letters must be capitalized in your describe. I think LOGIN PAGE WITH INCORRECT EMAIL ---NEGATIVE  , would be better.

File: modules/user/login/login_negativeEmptyPassword.spec.js [hunk 4]
```diff
@@ -1,30 +1,25 @@
 import { expect } from 'chai';
 import LoginPage from '../../_PageObjects/LoginPage';
+import {student} from  './../../_data/user.data';
+import Menu from "../../_PageObjects/Menu";
+
 
 describe('LOGIN PAGE NEGATIVE with valid Email and empty Password field', () => {
```
Comment (YelenaDipon) [inline modules/user/login/login_negativeEmptyPassword.spec.js]: All letters must be capitalized in your describe. I think LOGIN PAGE WITH VALID EMAIL AND EMPTY PASSWORD FIELD ---NEGATIVE , would be better.

File: modules/user/login/login_negativeInvalidEmail.spec.js [hunk 5]
```diff
@@ -1,33 +1,33 @@
 import { expect } from 'chai';
 import LoginPage from '../../_PageObjects/LoginPage';
+import {student} from  './../../_data/user.data';
+import Menu from "../../_PageObjects/Menu";
+import Notification from "../../_PageObjects/Notification";
 
 describe('LOGIN PAGE NEGATIVE with invalid Email and valid Password', () => {
```
Comment (YelenaDipon) [inline modules/user/login/login_negativeInvalidEmail.spec.js]: All letters must be capitalized in your describe. I think LOGIN PAGE WITH INVALID EMAIL AND VALID PASSWORD ---NEGATIVE , would be better.

File: modules/user/login/login_negativeEmptyEmail.spec.js [hunk 6]
```diff
@@ -1,29 +1,23 @@
...
-        const actual = LoginPage.h1.getText();
-        const expected = 'User Login';
-        expect(actual).eq(expected);
+        browser.waitUntil(() => Menu.h1.getText() === 'User Login',
+          5000, 'expected header to be different after 3s');
     });
 
     it('should fill out password field', () => {
-        LoginPage.password.setValue('123456');
+        LoginPage.passwordInput.setValue(123456);
```
[the remark this answers is not in the document]
Comment (Trivme) [reply modules/user/login/login_negativeEmptyEmail.spec.js]: Это wrong data - решил тут завести - они бльше нигде не используются

**Pull request merged**